### PR TITLE
Refactor/inline generator

### LINKED ISSUE #119: Refactor InlineGenerator -> Remove type hints in new class
Currently, the InlineGenerator adds tracing type hints & removes existing type hints. This is useful for evaluation. However, in production we only want to add the tracing type hints without changing existing type hints. Thus, a subclass of the corresponding AST Transformer should be created which does both, while the current AST Transformer only adds the tracing type hints.

### PR DESCRIPTION
**Pull request by RecurvedBow**
Closes #119

### CODE REVIEW

File: typegen/strats/__init__.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 from .gen import TypeHintGenerator
-from .inline import InlineGenerator
+from .inline import EvaluationInlineGenerator, InlineGenerator, RemoveAllTypeHintsTransformer
```
Comment (bengsparks) [inline typegen/strats/__init__.py:2]: Why export the `RemoveAllTypeHintsTransformer`?

File: typegen/strats/__init__.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 from .stub import StubFileGenerator
 
 __all__ = [
-    TypeHintGenerator.__name__, InlineGenerator.__name__, StubFileGenerator.__name__
+    TypeHintGenerator.__name__,
+    EvaluationInlineGenerator.__name__,
+    InlineGenerator.__name__,
+    StubFileGenerator.__name__,
+    RemoveAllTypeHintsTransformer.__name__
```
Comment (bengsparks) [inline typegen/strats/__init__.py:10]: Here too

**Review by bengsparks — CHANGES_REQUESTED** [review]

File: typegen/strats/__init__.py (hunk 1 above)
Comment (RecurvedBow) [reply typegen/strats/__init__.py]: Removed

File: typegen/strats/__init__.py (hunk 2 above)
Comment (RecurvedBow) [reply typegen/strats/__init__.py]: Removed

**Review by bengsparks — APPROVED** [review]

**Pull request merged**